app/src/main/java/com/edulectronics/tinycircuit/Views/CircuitActivity.java

```diff
@@ -312,8 +313,10 @@ private void giveNegativeFeedback(){
     //Create a positive feedback message
     private void givePositiveFeedback(){
         String[] positiveFeedback = getResources().getStringArray(R.array.positive_feedback);
+        String feedback = giveFeedback(positiveFeedback) + " " + getResources().getString(
```

Comment:
Dat idee klinkt logisch, maar er zijn wat problemen. Namelijk:
1 Er zijn resources nodig, die zijn alleen beschikbaar in een activity
2 Hij gebruikt een messagecontroller, die niet bestaat in de levelcontroller en wel in de circuitactivity. Twee van die dingen lijkt me ongewenst.